tests/AuthProvider/TokenTest.php

```diff
@@ -19,23 +19,21 @@ class TokenTest extends TestCase
 {
     public function testCreatingTokenAuthProvider()
     {
-        $options = $this->getOptions();
+        $options                     = $this->getOptions();
```

Comment:
I didn't find such spaces as part of PSR rules.
Could you please remove these spaces?

Reply:
Done, it was my editor aligning the `=`
But it's not in the PSR-2 so I removed it.